src/functions.php

```diff
@@ -736,3 +736,30 @@ function parse($value)
 {
     return Context::get()->getConfig()->parse($value);
 }
+
+function locateBinaryPath($name)
+{
+    $nameEscaped = escapeshellarg($name);
+
+    // Try `command`, should cover all Bourne-like shells
+    if (commandExist("command")) {
+        return run("command -v $nameEscaped")->toString();
+    }
+
+    // Try `which`, should cover most other cases
+    if (commandExist("which")) {
```

Comment:
Can I ask why? `command -v` is available for `zsh`, `bash`, `sh` and `fish` by default, which covers probably 99. percentile of systems out there. It is also a built-in for most implementations I know of, instead of external binary.